### DISCUSSION TITLE
Hybrid Networking Stack Demo

### DISCUSSION THREAD

**Discussion by maryamtahhan** [CloudNativeDataPlane/cndp#369] [Show and tell]
This Demo will showcase a CNDP-FRR vRouter an example of a hybrid network stack!

[Video](https://archive.fosdem.org/2023/schedule/event/hybrid_netstack/)
[Slides](https://archive.fosdem.org/2023/schedule/event/hybrid_netstack/attachments/slides/5741/export/events/attachments/hybrid_netstack/slides/5741/FOSDEM_23_Hybrid_Networking_Stack.pdf)

edit: added the video url

**Comment by KeithWiles** [comment]
HI Maryam, The video give me a 404 error, but slides look great.

**Reply by maryamtahhan** [reply]
Thanks for checking @KeithWiles I forgot to paste the actual link :$ #mortified

**Comment by shirangnihuakai** [comment]
<img width="1410" height="1242" alt="7bdfa29b-fec7-4a79-baac-adc86262a541" src="https://github.com/user-attachments/assets/9880fe32-9b11-4cbf-aed8-8a6f1e16746c" />
hello,In the traditional kernel network stack route, I saw a sidecar container, but I don't know what it is or what it's used for. Could you clarify this for me?
